### ISSUE TITLE
explicit hint to the tag list

### ISSUE THREAD

**Issue by nbehrnd** [openjournals/joss#1039]
May the documentation add a link to an explicit listing of all the tags used?

At present, searching publications in JOSS allows the criteria title, language, author, tag.  Inspired by Karl Voit's presentation at GLT18 ([45min video](https://www.youtube.com/watch?v=rckSVmYCH90)) including examples of tagging files locally and the benefits of this approach, and that there are publishers with public lists of the keywords they use to tag their publications (example [Wiley-VCH](https://application.wiley-vch.de/vch/journals/keyword.php)) with a set of keywords. Thus I think an explicit list of the tags JOSS uses would be an advantage.  It assists readers to query for publications of interest, authors to identify potential cross-sections of interest with publications already submitted.  It might be already in use internally if you assign which member/members of the Editorial Board does/do check publications about oceanography, or quantum chemistry.

**Comment by arfon** [comment]
Yes, this is a great idea! We're actually exploring something like this right now.

**Comment by nichtich** [comment]
See related or duplicate #677
